pkg/innerring/processors/container/process_container.go

```diff
@@ -14,10 +19,36 @@ func (cp *Processor) processContainerPut(put *containerEvent.Put) {
 		return
 	}
 
+	cnrData := put.Container()
+
+	// unmarshal container structure
+	// FIXME: temp solution, replace after neofs-api-go#168
+	cnrProto := new(containerGRPC.Container)
+	if err := proto.Unmarshal(cnrData, cnrProto); err != nil {
+		cp.log.Error("could not unmarshal container structure",
```

Comment:
What about logging it at `INFO` or `WARN` level? This is not an internal unhandled error in inner ring. Malformed container is a expected behavior and we process this normally.

Reply:
I agree, its better to have `INFO` lvl here.